test/helpers/mock-project.js

```diff
@@ -14,47 +16,94 @@ var readFile      = RSVP.denodeify(fs.readFile);
 
 function MockProject(root) {
   assert.equal(!!root, true, 'Must provide a root');
-  this.projectRoot = root;
+  this.projectRoot = root + '-' + Math.round(Math.random() * 10000000);
 }
 
 MockProject.prototype = {
```

Comment:
I think I will soon refactor this out into its own basic JavaScript lib, with the intent of allowing it to be a kind of Pretender for build tools and test scripts